tensorflow/core/ops/math_ops.cc

```diff
@@ -131,90 +131,102 @@ tf.complex_abs(x) ==> [5.25594902, 6.60492229]
 
 // Declares cwise unary operations signature: 't -> 't
 #define UNARY()                      \
-  Input("x: T").Output("y: T").Attr( \
-      "T: {half, float, double, int32, complex64, int64}")
+  Input("x: T").Output("y: T")
```

Comment:
Since you're touching the code, let's go ahead and add `complex128` support.  It should just mean adding it to the macro here and a few register macros in `cwise_op_*` as you say.  Thanks!

Your list of macros seems good, except that we should go ahead and allow `complex128` everywhere that `complex64` is allowed.